organization/_posts/2019/2019-05-29-packaging.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "HSF Packaging Meeting #34, 29 May, 2019"
+layout: meetings
+---
+
+# {{page.title}}
+
+## Agenda
+[<span class="underline">https://indico.cern.ch/event/819329/</span>](https://indico.cern.ch/event/819329/)
+
+Participants: Ben Morgan, Patrick Gartung, Graeme Stewart, Javier Cervantes, Chris Burr,
+Pere Mato, Dimitri, Geri Ganis, Guilherme Amadio, Marco Clemencic, Chris Green, Elizabeth Sexton-Kennedy
```

Comment:
To add "Ivan Razumov"
And Dimitri is "Dmitri Konstantinov"

I had to look up some people's names!